admin_guide/install/advanced_install.adoc

```diff
@@ -83,22 +83,71 @@ the remaining instructions.
 
 == Configuring Ansible
 
-You must create an *_/etc/ansible/hosts_* file, which is Ansible's inventory
-file for the playbook to use during the installation.
+The *_/etc/ansible/hosts_* file is Ansible's inventory
+file for the playbook to use during the installation. Replace the contents of the file with your desired configuration.
 
-Any hosts you designate as masters during the installation process should also
-be configured as
+[NOTE]
+====
+Before running the Ansible installer, any hosts you intend to designate as
+masters during the installation process should also be configured as
 link:../../admin_guide/manage_nodes.html#marking-nodes-as-unschedulable-or-schedulable[unschedulable]
-nodes. This is so the masters are configured as part of the
+nodes, in order to configure the masters as part of the
 link:../../architecture/additional_concepts/networking.html#openshift-sdn[OpenShift
 SDN].
+====
 
-The following sections describe example inventory files for various environment
-topographies, including configuring
-link:../../architecture/infrastructure_components/kubernetes_infrastructure.html#high-availability-masters[multiple
+The following sections describe example *_/etc/ansible/hosts_* files for various environment
+topographies, including example configurations for
+link:../../architecture/infrastructure_components/kubernetes_infrastructure.html#high-availability-masters[using multiple
 masters for high availability]. You can choose an example that matches your
-requirements to modify and use as your inventory file when
-link:#running-the-ansible-installer[running the Ansible installer].
+requirements and use as your inventory file when
+link:#running-the-ansible-installer[running the Ansible installer], or modify an example to match your own environment.
+
+*Configuring Node Host Labels*
+
+To assign link:../../architecture/core_concepts/pods_and_services.html#labels[labels] to node hosts during the Ansible install, indicate the desired labels in the *_/etc/ansible/hosts_* file:
+
+----
+[nodes]
+node1.example.com openshift_node_labels="{'region': 'primary', 'zone': 'east'}"
+----
+
+Labels are useful for determining the placement of pods onto nodes using the link:../scheduler.html#configurable-predicates[scheduler].
+
+*Configuring Host Variables*
+
+To assign environment variables to hosts during the Ansible install, indicate the desired variables in the *_/etc/ansible/hosts_* file:
+
+----
+[masters]
+ec2-52-6-179-239.compute-1.amazonaws.com
+openshift_public_hostname=ose3-master.public.example.com
```

Comment:
I assume any variables need to stay on the same line as the host entry. ^ @detiber @sdodson ?